package.json

```diff
@@ -11,29 +11,45 @@
   },
   "devDependencies": {
     "babel-core": "^6.25.0",
-    "babel-loader": "^7.0.0",
+    "babel-loader": "^7.1.1",
     "babel-plugin-transform-react-jsx": "^6.24.1",
     "babel-preset-env": "^1.5.2",
     "chrome-remote-interface": "^0.25.5",
     "compression-webpack-plugin": "~1.0.0",
     "copy-webpack-plugin": "~3.0.1",
     "css-loader": "^0.28.11",
-    "eslint": "^3.0.0",
-    "eslint-loader": "~1.6.1",
+    "eslint": "^3.19.0",
+    "eslint-loader": "^1.9.0",
     "eslint-plugin-react": "~6.9.0",
     "extract-text-webpack-plugin": "^2.1.0",
     "htmlparser": "^1.7.7",
     "jed": "^1.1.1",
-    "jshint": "~2.9.1",
-    "jshint-loader": "~0.8.3",
+    "jshint": "^2.9.5",
+    "jshint-loader": "^0.8.4",
+    "less": "^3.8.0",
+    "less-loader": "^4.1.0",
     "po2json": "^0.4.5",
     "sass-loader": "^7.0.3",
     "sizzle": "^2.3.3",
     "stdio": "^0.2.7",
     "webpack": "^2.6.1"
   },
   "dependencies": {
+    "bootstrap": "^4.1.1",
+    "bootstrap-less": "^3.3.8",
+    "create-react-class": "^15.6.3",
+    "exenv": "^1.2.2",
+    "jquery": "^3.3.1",
```

Comment:
Please avoid introducing jQuery in new code, and particularly not with react. I don't see it actually being used here? I think this can just be dropped. Likewise, please comb the list for other unused stuff, like "exenv" -- it just unnecessarily enlarges `npm install` time, and all of these dependencies introduce maintenance. `react-create-class` also sounds like dead weight -- use proper ES6 classes instead.

Reply:
I changed it. But dockerUI uses Id.